apps/sample-app/src/components/Calendar.tsx

```diff
@@ -1,21 +1,52 @@
 import './styles.css';
 
 import { Button } from '@fluentui/react-components';
-import { calendar } from '@microsoft/teams-js';
+import { app, calendar } from '@microsoft/teams-js';
 import React from 'react';
 
-export const Calendar: React.FunctionComponent = () => {
-  const handleCalendar = async (): Promise<void> => {
-    // create calendar paramaters
+import { MessageListItem } from './Emails';
+
+const teamsDeepLinkHost = 'teams.microsoft.com';
+const teamsDeepLinkProtocol = 'https';
+const teamsDeepLinkAttendeesUrlParameterName = 'attendees';
+const teamsDeepLinkUrlPathForCalendar = '/l/meeting/new';
+
+export const handleNewMail = async (): Promise<void> => {
+  if (!calendar.isSupported()) {
```

Comment:
I think what you were saying was that using calendar here is a workaround for mail capability not being enabled, is that right?

Reply:
I see why this is confusing, I should change "handleNewMail" to "handleNewMeeting". This functionality is independent of whether the "mail" capability is enabled. There are two different things happening - 1. "Mail" Capability from './utils' is to start a new email to one of the People Avatars (that is why it is meant to show up on the avatar hover menu alongside call, message functions). 2. "Add a new Meeting" comes from "Compose New Meeting" capability in "Calendar" to create a new meeting invite, without any specific person mentioned